docs/_news/news13.md

```diff
@@ -0,0 +1,70 @@
+- Open and view [project log files in the project overview page](https://github.com/eclipse/codewind/issues/3020).
+- In Eclipse 2020-03 (4.15) and later, you can [debug Node.js projects directly in Eclipse](https://github.com/eclipse/codewind/issues/2975). If preferred, use the Codewind preferences to go back to using a Chrome-based browser for debugging.
+- To learn more about why a push registry is required for remote Codewind style projects, [a link to the documentation is included in related dialogs](https://github.com/eclipse/codewind/issues/2992).
+- The Codewind Explorer view displays an indication of project errors and warnings. [Hover over the project to get the details](https://github.com/eclipse/codewind/issues/655).
+
+#### IntelliJ
+- The Codewind plug-in requires IntelliJ IDEA 2020.1.1 as the minimum version.
+
+##### VS Code
+- A [changelog](https://github.com/eclipse/codewind-vscode/wiki/Changelog) file is linked to the VS Code tools `README` file and is updated for Codewind 0.13.0.
+- The [`Codewind: Capture Diagnostics` command](https://github.com/eclipse/codewind/issues/2851) executes `cwctl diagnostics` to capture VS Code and Codewind logs. The `.zip` file that this command creates is uploaded with bug reports.
+- If you run a load test through the Performance Dashboard, [the load runner status appears in a VS Code notification](https://github.com/eclipse/codewind/issues/2739).
+- After you create a project, [the project logs appear as they become available](https://github.com/eclipse/codewind/issues/2755). If you want, you can also disable the log appearance by setting your preferences.
+- [Manage and view project logs that are currently available in the project overview page](https://github.com/eclipse/codewind/issues/2755).
+- A modified version of the [`Codewind: Manage Logs` command is available in Che](https://github.com/eclipse/codewind/issues/2755).
+- In the background, Codewind tries to [reconnect to a remote connection](https://github.com/eclipse/codewind/issues/2640#issuecomment-632896965) that it lost the connection to. If your network connection or a proxy connection is lost briefly but then restored, the connection reconnects after a short delay.
+- [If you delete an active project from the disk](https://github.com/eclipse/codewind/issues/2278), Codewind [deletes it automatically](https://github.com/eclipse/codewind/issues/2772).
+- Template source addition and removal is [reported immediately](https://github.com/eclipse/codewind/issues/2776).
+- [A warning appears if a project directory is added to multiple connections](https://github.com/eclipse/codewind/issues/2922). Adding a project directory to multiple connections is still allowed.
+- If you disable remote connections, they [remain disabled after you restart VS Code](https://github.com/eclipse/codewind/issues/2823).
```

Comment:
This is the same as the item `Codewind remembers the state of remote connections, whether connected or disconnected` under Eclipse bugfixes, in case you want to group them